README.md

```diff
@@ -33,29 +33,27 @@ Follow the [quickstart
 instructions](<https://python-for-android.readthedocs.org/en/latest/quickstart/>)
 to install and begin creating APKs.
 
-Quick instructions to start would be:
+**Quick instructions**: install python-for-android with:
 
     pip install python-for-android
 
-or to test the develop branch:
+(for the develop branch: `pip install git+https://github.com/kivy/python-for-android.git`)
 
-    pip install git+https://github.com/kivy/python-for-android.git
+Test that the install works with:
 
-The executable is called ``python-for-android`` or ``p4a`` (both are
-equivalent). To test that the installation worked, try::
+    p4a --version
 
-    python-for-android recipes
+To build any actual apps, **set up the Android SDK and NDK**
+as described in the [quickstart](
+<https://python-for-android.readthedocs.org/en/latest/quickstart/#installing-android-sdk>).
```

Comment:
Is this a valid markdown? Don't we need to remove the `<>` inside the parenthesis?

Reply:
I have no idea I copy&pasted that from above :joy: github seems to render it just fine though

Reply:
Thanks should be fine then :+1: